wgpu/src/image.rs

```diff
@@ -147,6 +155,7 @@ impl Pipeline {
                     depth_bias: 0,
```

Comment:
similarly, can omit the depth stuff if it's zero

Reply:
Done in b689778ed94b34706bf97e5994a721a8648386a5.